Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -175,6 +168,37 @@ - (void)urlSessionTask:(NSURLSessionTask *)sessionTask setState:(NSURLSessionTas
     [SentryLog logWithMessage:@"Finished HTTP span for sessionTask" andLevel:kSentryLevelDebug];
 }
 
+- (void)breadcrumbForSessionTask:(NSURLSessionTask *)sessionTask
```

Comment:
`l`: It totally makes sense to put this into an extra method. Maybe the following is a better method name.
```suggestion
- (void)addBreadcrumbForSessionTask:(NSURLSessionTask *)sessionTask
```